src/xmipp/libraries/py_xmipp/deepLearningToolkitUtils/utils.py

```diff
@@ -3,19 +3,21 @@
 
 BAD_IMPORT_MSG='''
 Error, tensorflow/keras is probably not installed. Install it with:\n  ./scipion installb deepLearningToolkit
-If gpu version of tensorflow desired, install cuda 8.0 or cuda 9.0
-We will try to automatically install cudnn, if unsucesfully, install cudnn and add to LD_LIBRARY_PATH
-add to SCIPION_DIR/config/scipion.conf
+If gpu version of tensorflow desired, modify  SCIPION_DIR/config/scipion.conf
 CUDA = True
-CUDA_VERSION = 8.0  or 9.0
-CUDA_HOME = /path/to/cuda-%(CUDA_VERSION)
-CUDA_BIN = %(CUDA_HOME)s/bin
-CUDA_LIB = %(CUDA_HOME)s/lib64
-CUDNN_VERSION = 6 or 7
-'''
 
+and 
+
+XMIPPP/xmipp.conf
```

Comment:
Also, this is not user friendly. After release, we should really focus on improving the set up